api_docs/message-formatting.md

```diff
@@ -0,0 +1,44 @@
+# Message formatting
+
+Zulip supports an extended version of Markdown for messages, as well as
+some HTML level special behavior. This article is currently a changelog
+for updates to these features.
```

Comment:
Let's prominently link to the `/help/` center page as the primary documentation for our markup, and mention that you can use https://zulip.com/api/render-message to get the current HTML version of any Markdown syntax.